docs/guides/managing_state.md

```diff
@@ -0,0 +1,154 @@
+
+## Setting Up Your Environment
+
+To get started with state management in Massdriver, follow these steps:
+
+### 1. Configure Your Environment Variables
+
+Set up your environment variables for authentication and state management:
+
+```bash
+
+export TF_HTTP_USERNAME=${MASSDRIVER_ORG_SLUG}
+export TF_HTTP_PASSWORD=${MASSDRIVER_SERVICE_ACCOUNT_TOKEN}
+
+# Your package friendly ID available from your details panel (without the four character suffix -xxxx)
+export MASSDRIVER_PACKAGE_ID="YOUR-PACKAGE-SLUG"
+
+# Massdriver supports putting multiple IaC tools into the same "bundle". 
+# Put the name of the step your terraform is in here.
+export MASSDRIVER_PACKAGE_STEP_NAME="the-step-name-in-your-package"
+
+export TF_HTTP_ADDRESS="https://api.massdriver.cloud/state/${MD_PKG_SLUG}/${MASSDRIVER_PACKAGE_STEP_NAME}"
+export TF_HTTP_LOCK_ADDRESS=${TF_HTTP_ADDRESS}
+export TF_HTTP_UNLOCK_ADDRESS=${TF_HTTP_ADDRESS}
+```
+
+### 2. Initialize Your Terraform Module
+
+Create a new `backend.tf` file with the following content to configure the HTTP backend:
+
+```bash
+echo 'terraform {
+  backend "http" {}
+}' > backend.tf
+```
+
+Initialize your Terraform module:
+
+```bash
+tofu init
+```
+
+### 3. Pull Existing State
+
+If you need to pull your state to inspect or edit:
+
+```bash
+tofu init
+tofu state pull > tofu.tfstate
+```
+
+### 5. Importing Resources into State
+
+To import an existing resource into state, you can use the `tofu import` command. This command is useful when you have existing infrastructure that was not created using OpenTofu / Terraform, but you want to manage it using automation going forward.
+
+The `tofu import` command allows you to specify the resource type and the resource ID. Importing a resource into Terraform state enables you to manage its lifecycle, apply changes, and track its state using Terraform commands and workflows.
+
+1. Identify the resource to be imported.
+2. Use the `tofu import` command to add the resource to your state:
+
+   ```bash
+   tofu import aws_instance.example i-1234567890abcdef0
+   ```
+
+3. Verify the resource has been imported:
+
+   ```bash
+   tofu state list
+   ```
+
+### 6. Removing resources from state
+
+The `terraform state rm` command is used to remove a resource from the Terraform state. This command is useful when you want to delete a resource that was previously imported or created using Terraform. However, it's important to note that if you remove a resource from the state, you should also update the corresponding code to avoid any conflicts or unintended changes.
```

Comment:
```suggestion
The [`tofu state rm`](https://opentofu.org/docs/cli/commands/state/rm/) command is used to remove a resource from the state file. This command is useful when you want to delete a resource that was previously imported or created using OpenTofu. However, it's important to note that if you remove a resource from the state, you should also update the corresponding code to avoid any conflicts or unintended changes.
```